README.md

```diff
@@ -21,10 +21,28 @@ Properties or `Item Asset` Fields to simplify items search. It may be extended w
 
 ## Item Properties or Asset Fields
 
-| Field Name           | Type                      | Description |
-| -------------------- | ------------------------- | ----------- |
-| hsi:wavelength_min   | \[number]                 | **REQUIRED**. Min center wavelength of the item, in micrometers (μm) |
-| hsi:wavelength_max   | \[number]                 | **REQUIRED**. Max center wavelength of the item, in micrometers (μm) |
+| Field Name         | Type      | Description                                                          |
+| ------------------ | --------- | -------------------------------------------------------------------- |
+| hsi:wavelength_min | \[number] | **REQUIRED**. Min center wavelength of the item, in micrometers (μm) |
+| hsi:wavelength_max | \[number] | **REQUIRED**. Max center wavelength of the item, in micrometers (μm) |
+
+## Media Types
+
+This section describes the recommended media types to use for various hyperspectral
+image formats.
+
+| Name                          | Media Type                                               | Notes              |
+| ----------------------------- | -------------------------------------------------------- | ------------------ |
+| GeoTIFF                       | image/tiff; application=geotiff                          |                    |
+| Cloud-optimized GeoTIFF (COG) | image/tiff; application=geotiff; profile=cloud-optimized |                    |
+| HDF                           | application/x-hdf                                        |                    |
+| HDF5                          | application/x-hdf5                                       |                    |
+| NetCDF                        | application/netcdf                                       |                    |
+| Zarr                          | application/vnd.zarr                                     |                    |
+| ENVI Data                     | application/vnd.nv5.envi-data                            | New recommendation |
+| ENVI Header                   | application/vnd.nv5.envi-header                          | New recommendation |
```

Comment:
Are these officially registered mime types?

Reply:
they aren't, that's why they're vnd